src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -0,0 +1,150 @@
+    const formattedTokenValue = formatterUtils.formatNumber(amount, {
+        format: NumberFormat.TOKEN_AMOUNT_SHORT,
+        withSign: true,
+        fallback: '-',
+    });
+    const fiatValue = Number(amount) * Number(fiatPrice);
+    const formattedFiatValue = formatterUtils.formatNumber(fiatValue, {
+        format: NumberFormat.FIAT_TOTAL_SHORT,
+        fallback: ` `,
+    });
+    const formattedTokenAmount = `${formattedTokenValue} ${tokenSymbol}`;
+
+    return { formattedTokenAmount, formattedFiatValue };
+};
+
+export const AssetTransfer: React.FC<IAssetTransferProps> = (props) => {
+    const {
+        senderAddress,
+        recipientAddress,
+        senderEnsName,
+        recipientEnsName,
+        tokenName,
+        tokenIconSrc,
+        amount,
+        symbol,
+        fiatPrice,
+        chainId,
+        hash,
+        wagmiConfig: wagmiConfigProps,
+    } = props;
+
+    const wagmiConfigProvider = useConfig();
+
+    const wagmiConfig = wagmiConfigProps ?? wagmiConfigProvider;
+
+    const processedChainId = chainId ?? wagmiConfig.chains[0].id;
+
+    const currentChain = wagmiConfig.chains.find(({ id }) => id === processedChainId);
+    const blockExplorerUrl = currentChain?.blockExplorers?.default.url;
+
+    const blockExplorerAssembledHref = blockExplorerUrl ? `${blockExplorerUrl}/tx/${hash}` : undefined;
+
+    const { formattedTokenAmount, formattedFiatValue } = formatValue(amount, symbol, fiatPrice);
+
+    const tokenTransferClassNames = classNames(
+        'flex h-16 w-full items-center justify-between rounded-xl border-[1px] border-neutral-100 px-4', // base
+        'hover:border-neutral-200 hover:shadow-neutral-md', // hover
+        'focus:outline-none focus-visible:rounded-xl focus-visible:ring focus-visible:ring-primary focus-visible:ring-offset', // focus
+        'active:border-neutral-300', // active
+        'md:h-20 md:px-6', // responsive
+    );
+
+    return (
+        <div className="flex h-full w-[320px] flex-col gap-y-2 md:w-[640px] md:gap-y-3">
+            <div className="relative flex h-full  flex-col rounded-xl border-[1px] border-neutral-100 md:flex-row">
+                <AssetTransferAddress
+                    txRole="sender"
+                    ensName={senderEnsName}
+                    address={senderAddress}
+                    blockExplorerUrl={blockExplorerUrl}
+                />
+                <div className="border-t-[1px] border-neutral-100 md:border-l-[1px]" />
+                <AvatarIcon
+                    icon={IconType.CHEVRON_DOWN}
+                    size="sm"
+                    className={classNames(
+                        'absolute left-4 top-1/2 -translate-y-1/2 bg-neutral-50 text-neutral-300', //base
+                        'md:left-1/2 md:-translate-x-1/2 md:-rotate-90', //responsive
+                    )}
+                />
+                <AssetTransferAddress
```

Comment:
The asset-transfer-address and asset-transfer-amount components are missing the white background color. Also double check the hover and active states, they look wrong